base/pkg/entry.jl

```diff
@@ -376,7 +376,11 @@ function update(branch::AbstractString)
                         LibGit2.fetch(repo)
                         LibGit2.merge!(repo, fastforward=true)
                     catch err
-                        show(err)
+                        if isa(err, LibGit2.GitError)
+                            print("Package $pkg cannot be updated. Error: $(err.msg) Class: $(err.class)")
+                        else
+                            show(err)
```

Comment:
Yes, multiple people other than me have said you're catching too many errors here. The way in which the LibGit2 module is coded is bug-prone and needs to be fixed.